Apps/Admin/Client/Pages/UserInfo.razor

```diff
@@ -0,0 +1,30 @@
+@page "/userinfo"
```

Comment:
We should use kebab case for URLs: `/user-info`.

Reply:
fixing